### PR TITLE
Allow to alter zscaler IPs list

### PR DESCRIPTION
**Pull request by PiotrPalkaSpotOn**
- default ZScaler Proxy IPs works well in most AWS VPCs but not everywhere
- we have to have a way to override them

### CODE REVIEW

**Comment by dalefrancum** [comment]
@PiotrPalkaSpotOn - Can you update the chart version?

**Comment by PiotrPalkaSpotOn** [comment]
@dalefrancum right... Always forget about this one. Updated. Thanks for a good eye!

File: monochart/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.20
+version: 1.1.21
```
Comment (dalefrancum) [inline monochart/Chart.yaml:18]: Good as of right now, just need to see which one of ours gets merged first. I'll let you know if mine does.

File: monochart/templates/ingress.yaml [hunk 2]
```diff
@@ -43,14 +43,16 @@ metadata:
...
         "values":[
-          "34.238.231.165/32",
-          "34.202.59.188/32",
-          "34.197.162.237/32"
+          {{- /* get first element of ZScaler proxy IP list. It doesn't require a comma */ -}}
+          {{- first $ingress.zscalerProxyIPs | quote -}}
+          {{- /* get other elements. They will require comma before the value */ -}}
+          {{- range ( rest $ingress.zscalerProxyIPs ) -}}
+            , {{- . | quote -}}
+          {{- end -}}
```
Comment (dalefrancum) [inline monochart/templates/ingress.yaml:55]: Wow, nice template work.

**Review by dalefrancum — APPROVED** [review]

File: monochart/Chart.yaml (hunk 1 above)
Comment (dalefrancum) [reply monochart/Chart.yaml:18]: Okay, mine did get merged, so you need to update and bump to 1.2.22. Thanks.

Comment (PiotrPalkaSpotOn) [reply monochart/Chart.yaml:18]:
```suggestion
version: 1.1.22
```

**Pull request merged**